src/main.c

```diff
@@ -192,9 +195,9 @@ typedef struct{
   uint32_t http_control_server_enable;
   uint32_t avoid_overlaping_touchscreen;
   float custom_font_scale;
-  uint32_t hardcore_mode; 
+  ra_config_t ra_config;
```

Comment:
Only uint32_t and floats are allowed in this struct (no nested structures). This is to make sure the structure stays compatible across builds with different compilers.